js/scripts.js

```diff
@@ -0,0 +1,24 @@
+var a=-7;
+    b = -8;
+    value = ((a*a) + (2*a*b) - (b-b));
+
+console.log(value);
+
+
+if (value < 0) {
+    console.log("wynik ujemny")
+  } 
+else if (value=0) {
```

Comment:
else i elseify zaczynamy na tej samej linii, np:

```
if (warunek) {
    // do smth
} else {
    // do smth else
}
```

w `else if` nie porównujesz do zera tylko nadajesz value wartość zera. `=` to przypisanie wartości, porównanie to `===` - swoją drogą brak spacji przed i po `=`